openmls/src/group/mls_group/create_commit_params.rs

```diff
@@ -2,12 +2,20 @@
 
 use super::{proposals::ProposalStore, *};
 
+/// Can be used to denote the type of a commit.
+#[derive(Debug, Copy, Clone)]
+pub enum CommitType {
+    External,
+    Internal,
```

Comment:
So far we don't use the name "internal" anywhere. I'm not super hung up on it, but in general the opposite of "external" has been "member", also in the spec. I'd vote for changing it to that.